### PR TITLE
feat(molecule/notification): add new variation property for positive colors

### PR DESCRIPTION
**Pull request by midudev**
**This PR implements version 2.1 for Molecule Notification.**

Create new prop `variation` to add positive colors to the types. So now you could have negative (bold colors) by default, and a positive one with colors washed out.

It keeps compatibility with the previous version as the default prop for `variation` is `negative`.

👀 [Demo](https://sui-components-pr-molecule-notification-2-1.now.sh/workbench/molecule/notification/demo)
🎨   [Paper](https://paper.dropbox.com/doc/SUI-Notifications-nM2P1iTfKFUlPZ3GPGUoI)

image

### CODE REVIEW

**Review by danivm — DISMISSED** [review]

**Comment by carlosvillu** [comment]
I miss here enums with the possibles values for each prop.

**Review by carlosvillu — DISMISSED** [review]

File: components/molecule/notification/src/index.js [hunk 1]
```diff
@@ -28,6 +28,11 @@ const AUTO_CLOSE_TIME = {
 const TRANSITION_DELAY = 1000 // ms
 const BUTTONS_MAX = 3 // buttons
 
+const VARIATIONS = {
+  negative: 'negatvie',
```
Comment (JoanClaret) [inline components/molecule/notification/src/index.js]: typo error `negatvie` => `negative`

Comment (midudev) [reply components/molecule/notification/src/index.js]: Magic string ftw! Fixed, thanks!

**Review by jcger — APPROVED** [review]

**Review by JoanClaret — APPROVED** [review]
good job dude!

**Review by danivm — APPROVED** [review]

**Pull request merged**